charts/scalardb-cluster/templates/scalardb-cluster/validate-values.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if .Values.scalardbCluster.podAnnotations }}
+  {{- if hasKey .Values.scalardbCluster.podAnnotations "checksum/config" }}
+    {{- fail "ScalarDB Cluster: \"scalardbCluster.podAnnotations\" cannot contain the key \"checksum/config\" as it is managed by the chart." }}
+  {{- end }}
+{{- end }}
+
+{{- if .Values.scalardbCluster.podLabels }}
+  {{- if hasKey .Values.scalardbCluster.podLabels "azure-extensions-usage-release-identifier" }}
+    {{- fail "ScalarDB Cluster: \"scalardbCluster.podLabels\" cannot contain the key \"azure-extensions-usage-release-identifier\" as it is managed by the chart." }}
+  {{- end }}
+{{- end }}
```

Comment:
This chart automatically sets (manages) specific names' keys. We want to prevent users from overriding such keys. So, I added validations for them.

Reply:
This validation was added based on the review comment in another PR.
- https://github.com/scalar-labs/helm-charts/pull/306#discussion_r2290397335